script/plugin_webpack_directories.rb

```diff
@@ -39,8 +39,10 @@ def entry_name(plugin_name, entry)
 specs.each do |dep|
   # skip other rails engines that are not plugins
   # TODO: Consider using the plugin registration api?
-  next unless dep.name =~ plugin_name_regexp
-  next if dep.name =~ /.*[_-]core$/
+  unless dep.respond_to?("metadata") && dep.metadata.has_key?("is_foreman_plugin") && dep.metadata["is_foreman_plugin"] == "true"
+    next unless dep.name =~ plugin_name_regexp
+    next if dep.name =~ /.*[_-]core$/
```

Comment:
Aren't the `[_-]core` plugin on their way out?

Reply:
They should be out.